linkerd/protocol/thrift/src/main/scala/io/buoyant/linkerd/protocol/ThriftInitializer.scala

```diff
@@ -19,15 +19,9 @@ class ThriftInitializer extends ProtocolInitializer {
   protected val defaultServer = Thrift.server
     .configured(Server.Port(4114))
 
-  val Framed = Parsing.Param.Boolean("thriftFramed") { framed =>
-    Thrift.param.Framed(framed)
-  }
```

Comment:
Whoops, yep, I was using the wrong snapshot in my testing and didn't pick this up :(